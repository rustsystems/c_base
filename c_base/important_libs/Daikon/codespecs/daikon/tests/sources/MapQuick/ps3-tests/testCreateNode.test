#Test creating a node
CreateNode n1 2
