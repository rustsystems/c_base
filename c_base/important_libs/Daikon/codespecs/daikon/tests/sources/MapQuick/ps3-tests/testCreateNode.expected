#Test creating a node
created node n1 with cost 2
